src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task {
+    public static final char TASK_TYPE = 'D';
```

Comment:
Consider removing the above variable and instead incorporating it into the toString() method 

example: 
return "[D]" + ...

Reply:
There is nothing wrong in declaring a constant for specific keywords. I think this is fair enough.
Using `"[D]"` results in the code reader asking the question: why is it `"[D]"` and not any other letter?